### PR TITLE
e2e test for SUT debugging

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts [hunk 1]
```diff
@@ -183,6 +183,36 @@ module.exports = {
...
+      .waitForElementVisible('*[id="singleTesttests/4_Ballot_test.sol"]', 60000)
+      .click('*[id="singleTesttests/4_Ballot_test.sol"]')
+      .click('#runTestsTabRunAction')
+      .waitForElementVisible('*[data-id="testTabSolidityUnitTestsOutputheader"]', 120000)
+      .waitForElementContainsText('#solidityUnittestsOutput', 'tests/ballotFailedDebug_test.sol', 60000)
+      .waitForElementContainsText('#solidityUnittestsOutput', '✘ Check winning proposal', 60000)
+      .waitForElementContainsText('#solidityUnittestsOutput', '✓ Check winnin proposal with return value', 60000)
+      .click('.fa-bug')
+      .waitForElementContainsText('*[data-id="sidePanelSwapitTitle"]', 'DEBUGGER', 60000)
+      .waitForElementContainsText('*[id="FunctionPanel"]', 'checkWinningProposal()', 60000)
```
Comment (yann300) [inline apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts:200]: if possible it should only use `data-id`

Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts:200]: for now, there is no data-id assigned to that section

Comment (yann300) [reply apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts:200]: probably that that should be added then

**Review by yann300 — APPROVED** [review]

File: apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts (hunk 1 above)
Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts:200]: yes, will do

Comment (Aniket-Engg) [reply apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts]: done

**Comment by yann300** [comment]
LGTM

**Pull request merged**